lib/services/sorting/post/post_score_functions.dart

```diff
@@ -22,15 +22,17 @@ double voteScore(PostFirestore post, GeoPoint userPosition) {
 /// it to -postDate since the same constant added to all scores (currentDate)
 /// does not matter. See [PostSortOption] constructor documentation for more
 /// details.
-double dayScore(PostFirestore post, GeoPoint userPosition) {
+double dayScore(PostFirestore post, GeoPoint _) {
```

Comment:
For unused parameters, does it work to simply prefix the argument name by an underscore instead of completely stripping out the name altogether ?

For example, here we could use `_userPosition` so it is still clear what the argument refers to.

If it works, could you apply this suggestion to all unused parameters, please ?

Reply:
When I try to apply your suggestion, I get a warning. Do we keep it like that then?

Reply:
No, then keep it the old way, I was wrong.